pkgs/development/libraries/tdlib/default.nix

```
@@ -1,14 +1,20 @@
 { fetchFromGitHub, gperf, openssl, readline, zlib, cmake, stdenv }:
 
 stdenv.mkDerivation rec {
-  version = "1.6.0";
+  version = "1.6.9";
```

Since there is no tag and https://github.com/tdlib/td/commit/32f2338bd199dd06a1b4b5f1ad14f2d4f2868f01 is just a random commit
```suggestion
  version = "unstable-2020-10-25";
```